File: app/src/main/java/com/safframework/app/MainActivity.java
package com.safframework.app;

import android.app.Activity;
import android.os.Bundle;

import com.safframework.utils.SAFUtils;

/**
 * Created by tony on 2017/2/26.
 */

public class MainActivity extends Activity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SAFUtils.makeLogTag(this.getClass());
    }
}

File: app/src/main/java/com/safframework/app/MainActivity.kt
package com.safframework.app

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.safframework.log.L
import com.safframework.utils.getDeviceUsableMemory
import com.safframework.utils.getSDKVersion

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        L.i("可用内存：" + getDeviceUsableMemory(this))
        L.i("sdk版本：" + getSDKVersion())
    }
}
